libs/pjproject/patches/150-config_site.patch

```diff
@@ -11,34 +11,23 @@
 + * Since both pjproject and asterisk source files will include config_site.h,
 + * we need to make sure that only pjproject source files include asterisk_malloc_debug.h.
 + */
-+
-+/* #if defined(MALLOC_DEBUG) && !defined(_ASTERISK_ASTMM_H)
-+ * #include "asterisk_malloc_debug.h"
-+ * #endif
-+ */
++#if 0 && !defined(_ASTERISK_ASTMM_H)
++#include "asterisk_malloc_debug.h"
++#endif
 +
 +/*
 + * Defining PJMEDIA_HAS_SRTP to 0 does NOT disable Asterisk's ability to use srtp.
 + * It only disables the pjmedia srtp transport which Asterisk doesn't use.
 + * The reason for the disable is that while Asterisk works fine with older libsrtp
 + * versions, newer versions of pjproject won't compile with them.
 + */
-+
-+/*
-+ * This doesn't disable SRTP completely, so we have to keep using the external
-+ * libsrtp, otherwise pjsip would just build the internal one.
-+ */
-+
 +#define PJMEDIA_HAS_SRTP 0
 +
-+/*
-+ * Defining PJMEDIA_HAS_WEBRTC_AEC to 0 does NOT disable Asterisk's ability to use
-+ * webrtc.  It only disables the pjmedia webrtc transport which Asterisk doesn't use.
-+ */
-+#define PJMEDIA_HAS_WEBRTC_AEC 0
-+
 +#define PJ_HAS_IPV6 1
++#if !defined(AST_DEVMODE) && !defined(PJPROJECT_BUNDLED_ASSERTIONS)
```

Comment:
This should not be in here, this is standalone pjsip, not from within asterisk. AST_DEVMODE shouldn't be referenced.

Reply:
Its a preprocessor define, nothing is referenced. Since standalone pjproject doesn't know anything about AST_DEVMODE the compile won't ever see that block.

Reply:
But why the extra clutter? Let's just keep NDEBUG unconditionally.